From d12926b84fa51824f6544bbcff768098fe70e15f Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Fri, 18 Aug 2017 11:32:33 -0500
Subject: [PATCH 2420/5242] MLK-16226: arm64: dts: disable the NAND node in
 i.MX8QXP default device tree

commit  8cbc3a3c95ca765dacd6ab70e208a8cebb4bedd1 from
https://source.codeaurora.org/external/imx/linux-imx.git

NAND node was disabled in default device tree, the patch 4ad48e3cf
wrongly inserted usb node in the middle caused NAND enabled.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 5817ef5..9cce368 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1376,6 +1376,7 @@
 		dmas = <&dma_apbh 0>;
 		dma-names = "rx-tx";
 		power-domains = <&pd_conn_nand>;
+		status = "disabled";
 	};
 
 	usbphynop1: usbphynop1 {
-- 
1.7.9.5

